src/results/ResultsContainer.tsx

```diff
@@ -273,19 +279,38 @@ export class Results extends Component<ResultsProps, ResultsContainerState> {
           }
           content={
             results.length ? (
-              <ResultsTable
-                results={results}
-                columnNames={columns}
-                handleRowSelect={this.handleRowSelect}
-                selectedRows={selectedRows}
-                handleHeaderClick={this.updateColumnSort}
-                sortColumn={sortColumn}
-                sortDirection={sortDirection}
-                handleLoadMoreRows={() =>
-                  dispatchFetchBatchOfResultsIfNeeded(nextUrl)
-                }
-                totalNumberResults={totalNumberResults}
-              />
+              <Fragment>
+                <div className="button-group">
+                  <button className="button link-button disabled">Blast</button>
+                  <button className="button link-button disabled">Align</button>
+                  <button className="button link-button">
+                    <DownloadIcon />
+                    Download
+                  </button>
+                  <button className="button link-button disabled">
+                    <BasketIcon />
+                    Add
+                  </button>
+                  <button className="button link-button">
+                    <StatisticsIcon />
+                    Statistics
+                  </button>
+                  <button className="button link-button">Map to</button>
+                </div>
+                <ResultsTable
+                  results={results}
+                  columnNames={columns}
+                  handleRowSelect={this.handleRowSelect}
```

Comment:
Since this is not wrapped inside an arrow function to capture `this`, it may cause some problems in some situations due to the `this` value being different than `this` of the instance of the class at the call time or inside the invoked method.

As `.bind()` is less popular these days, I think putting this inside an arrow function would make it a little bit more robust.

Reply:
This should be ok as both `handleRowSelect` and `updateColumnSort` are defined as arrow functions (see https://daveceddia.com/avoid-bind-when-passing-props/#way-4-property-initializers-good-esnext)

Reply:
Perfect, I totally missed that!